enclave/core/sgx/linux/threadlocal.c

```diff
@@ -204,16 +204,15 @@ static volatile bool _thread_locals_relocated = false;
 static __thread oe_tls_atexit_t* _tls_atexit_functions;
 static __thread uint64_t _num_tls_atexit_functions;
 
-/**
- * Get the address of the FS segment given a thread data object.
- * Currently FS is assumed to exist one page after the thread data.
- * This needs to be made more flexible, taking into account the
- * actual size of the tls data.
- */
+// TODO: Make this flexible in case more than one page of thread local storage
```

Comment:
Is this related to https://github.com/openenclave/openenclave/issues/1161?

Reply:
Yes thanks for your confirmation of my comments.